src/Parser/CommandInfo.php

```diff
@@ -334,6 +339,25 @@ public function getTopics()
     }
 
     /**
+     * Returns an array of commands that this command calls.
+     *
+     * @return string[]
+     */
+    public function getCalls()
+    {
+        return $this->calls;
+    }
+
+    public function setCalls($calls) {
+        if (is_string($calls)) {
+            $calls = explode(',', static::convertListToCommaSeparated($calls));
+        }
+        $this->calls = array_filter($calls);
```

Comment:
Maybe array-merge these together, so folks can use multiple `@calls` in addition to csv, if they prefer?